scripts/bootstrap

```diff
@@ -81,7 +81,7 @@ do
       ci=true ;;
     -t|--tgz_package)
       tgz_file=$2
-      if [ ! -f ${tgz_file} ]
+      if [[ ! -f ${tgz_file} ]]
```

Comment:
This one is working like this for me: `if [ ! -f ${tgz_file} ]`

Reply:
Not on Debian devices or VM